hash/msx2_flop.xml

```diff
@@ -14010,6 +14200,51 @@ The following floppies came with the machines.
 		</part>
 	</software>
 
+	<software name="trilogy">
+		<description>Trilogy Kuki Youka Shinden (Japan)</description>
+		<year>1990</year>
+		<publisher>HARD</publisher>
+		<info name="alt_title" value="Ｔｒｉｌｏｇｙ九鬼妖華真伝"/>
+		<part name="flop1" interface="floppy_3_5">
+			<feature name="part_id" value="Disk A"/>
+			<dataarea name="flop" size="737280">
+				<rom name="trilogy kuki youka shinden - hard (disk 1).dsk" size="737280" crc="bd3d7788" sha1="17313c306bd48dbfbaa0f45d0612626e4792747e"/>
+			</dataarea>
+		</part>
+		<part name="flop2" interface="floppy_3_5">
+			<feature name="part_id" value="Disk B"/>
+			<dataarea name="flop" size="737280">
+				<rom name="trilogy kuki youka shinden - hard (disk 2).dsk" size="737280" crc="b50c54d3" sha1="9f05a64f78e329ce855d92468a230673bfb3629c"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="tucs">
+		<description>The Tucs (Japan)</description>
+		<year>1988</year>
+		<publisher>MSX Magazine</publisher>
+		<info name="alt_title" value="ザ・タックス"/>
+		<info name="usage" value="Requires a Japanese system."/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="the tucs - msx magazine.dsk" size="737280" crc="e0cdb9c0" sha1="b219f5236d1a20f9d9b32dae1b4c86c70faf6b72"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="tulipich">
+		<description>Tulip Ichigo (Japan)</description>
+		<year>1991</year>
+		<publisher>Tempin Software</publisher>
+		<info name="alt_title" value="チューリップ壱号"/>
```

Comment:
If you’re going to be consistent in indicating presence of long vowels, it should be “Tulip Ichigou” (or “Tulip Ichigō”, the MSX software lists seem to use a mixture).

Reply:
Updated the description.